bach/bach/series/series_string.py

```diff
@@ -226,3 +228,66 @@ def __add__(self, other) -> 'Series':
 
     def _comparator_operation(self, other, comparator, other_dtypes=tuple(['string'])) -> 'SeriesBoolean':
         return super()._comparator_operation(other, comparator, other_dtypes)
+
+    def to_json_array(self, partition: WrappedPartition = None) -> 'SeriesJson':
```

Comment:
Also, we should add a check to know if the partition is correct.
```suggestion
    def to_json_array(self, window: Optional[WrappedWindow] = None) -> 'SeriesJson':
```

Reply:
Undid this change again, as I don't think this is a Window function. See other comment